types/detox/test/detox-jest-setup-tests.ts

```diff
@@ -2,7 +2,8 @@ declare var beforeAll: (callback: () => void) => void;
 declare var beforeEach: (callback: () => void) => void;
 declare var afterAll: (callback: () => void) => void;
 
-import adapter from "detox/runners/jest/adapter";
+import * as detox from "detox";
+import * as adapter from "detox/runners/jest/adapter";
```

Comment:
These should be `import detox = require("detox");` et al

Reply:
@Jessidhia fixed for the Jest test file; should I change the mocha setup test as well?